.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+python:
+    - "2.7"
+before_install:
+    - "export DISPLAY=:99"
+    - "sh -e /etc/init.d/xvfb start"
+install:
+    - "pip install -e git://github.com/edx/xblock-sdk.git#egg=xblock-sdk"
+    - "pip install -r requirements.txt"
+    - "pip install -r test-requirements.txt"
+    - "pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements.txt"
+    - "pip install -r $VIRTUAL_ENV/src/xblock-sdk/test-requirements.txt"
```

Comment:
@antoviaque just rebuilt my env using this commands - it worked and didn't throw an exception you faced. I'm committing it to the branch under review to see if travis will complain about it.